config/cesm/machines/config_machines.xml

```diff
@@ -1375,7 +1375,7 @@ This allows using a different mpirun command to launch unit tests
     <DIN_LOC_ROOT_CLMFORC>/project/tss</DIN_LOC_ROOT_CLMFORC>
     <DOUT_S_ROOT>/scratch/cluster/$USER/archive/$CASE</DOUT_S_ROOT>
     <BASELINE_ROOT>/fs/cgd/csm/ccsm_baselines</BASELINE_ROOT>
-    <CCSM_CPRNC>/fs/cgd/csm/tools/cime/tools/cprnc/cprnc.izumi</CCSM_CPRNC>
```

Comment:
@fischer-ncar  I think this will break cprnc on hobart, since now it looks like they are both using the same executable. I don't use hobart anymore myself, but this should be fixed if anyone does use hobart for testing.

Reply:
I just tried it on hobart and it seems to work fine.